src/js/components/Dashboard.js

```diff
@@ -204,7 +204,9 @@ class Dashboard extends Component {
       <div ref="content" className="dashboard">
         <Header direction="row" justify="between" large={true}
           pad={{horizontal: 'medium', between: 'small'}}>
-          {title}
+          <div>
```

Comment:
thanks, and we add an inline JSX comment to explain why the div is here?

I just want to make sure no one will remove this wrapper thinking it will not affect the results.